packages/gatsby-source-wordpress/src/gatsby-browser.ts

```diff
@@ -62,20 +74,13 @@ function hydrateImages(): void {
             hydrationData.innerHTML
           )
 
-          // @ts-ignore - createRoot is on ReactDOM
           if (ReactDOM.createRoot) {
-            // @ts-ignore - createRoot is on ReactDOM
-            const root = ReactDOM.createRoot(image.parentNode.parentNode)
```

Comment:
image.parentNode.parentNode is part of what was breaking this. That was the wrong element

Reply:
Removing the second `parentNode` in `image.parentNode.parentNode` results in a duplication of the `.gatsby-image-wrapper`-div for me.

Reply:
hm, perhaps we need to do a check there to see if it exists before using it 🤔 @Haldaug do you have a public repo where I can see that happening? Or maybe the contents of the html field from that page? I'm not sure why that wasn't happening for me while I was testing

Reply:
You can see the html markup here: https://build-018acaec-e7e7-4635-9322-dc4340659c72.gtsb.io/meninger-og-analyse/kommentar/nytt-sesongvarsel-med-hovedvekt-pa-juli-2022/

Reply:
Hi @Haldaug, sorry for the inconvinience here. Can you try `gatsby-source-wordpress@6.18.0-alpha-wordpress-image-hydration.29` which should fix the issue. PR here https://github.com/gatsbyjs/gatsby/pull/36002

Reply:
Hi @TylerBarnes. I can confirm that this fix resolved the issue. Will this be rolled out in the next version of gatsby-source-wordpress?

Reply:
@Haldaug , yes it will go out with the next OSS release which happens every 2 weeks. I believe this PR missed the cutoff so it will be out in the next release